smart-contracts/.prettierrc

```diff
@@ -27,7 +27,8 @@
         "tabWidth": 2,
         "useTabs": false,
         "singleQuote": false,
-        "semi": false
+        "semi": true,
```

Comment:
This should stay false, and fixing this will remove a bunch of other changes in this PR

Reply:
If false it will remove all semicolons, I think we want semicolons in ts?

Reply:
Let's do this in another PR.